packages/wallet/src/account.ts

```diff
@@ -33,7 +33,7 @@ import {
 } from './utils';
 
 type TxParamsType = Pick<TransactionRequestLike, 'gasLimit' | 'gasPrice' | 'maturity'>;
-
+export type AccountResource<T extends Account> = Resource & { account: ThisType<T> };
```

Comment:
I was thinking of defining this where `Resource` is defined (in the `providers` package), but that's not the place for it as `AccountResource` is related to accounts specifically.